### PR TITLE
[feature] provide `OwnerName` field for README template

### PR DESCRIPTION
**Pull request by everhopingandwaiting**
reopen: pr  #10356

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
I think we net do ducument the vars ... I'll send a pull for this

**Comment by 6543** [comment]
@everhopingandwaiting since #10983 got merged:
can you "update" your featurebranc and extend `docs/content/doc/advanced/customizing-gitea.en-us.md` by adding your new variable to the note?

**Comment by 6543** [comment]
@everhopingandwaiting a hint: you should make `git pull -r`  before `git push`

**Comment by everhopingandwaiting** [comment]
> @everhopingandwaiting a hint: you should make `git pull -r` before `git push`

should I choose rebase or merge to select upstream changes in my origin branch?
image

**Comment by 6543** [comment]
@everhopingandwaiting If you like to chat with me -> https://discord.gg/Gitea

**Comment by everhopingandwaiting** [comment]
> @everhopingandwaiting since #10983 got merged:
> can you "update" your featurebranch and extend `docs/content/doc/advanced/customizing-gitea.en-us.md` by adding your new variable to the note?

```
**NOTE:** readme templates support **variable expansion**.  
currently there are `{Name}` (name of repository), `{Description}`, `{CloneURL.SSH}` and `{CloneURL.HTTPS}`

```
seems I should change `OwnerName` to `Name`   😉

```go
"Description":    repo.Description,
		"CloneURL.SSH":   cloneLink.SSH,			"CloneURL.SSH":   cloneLink.SSH,
		"CloneURL.HTTPS": cloneLink.HTTPS,			"CloneURL.HTTPS": cloneLink.HTTPS,
		"OwnerName":      repo.OwnerName,
```

**Review by guillep2k — APPROVED** [review]

**Review by 6543 — CHANGES_REQUESTED** [review]
blocking since need addition to docs

**Comment by 6543** [comment]
just add your `OwnerName` to the list in `docs/content/doc/advanced/customizing-gitea.en-us.md`

**Comment by everhopingandwaiting** [comment]
> @everhopingandwaiting If you like to chat with me -> https://discord.gg/Gitea

Unknown cause ， the site is blocked 😜
image

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ Locales may change between versions, so keeping track of your customized locales
 To add a custom Readme, add a markdown formatted file (without an `.md` extension) to `custom/options/readme`
 
 **NOTE:** readme templates support **variable expansion**.  
-currently there are `{Name}` (name of repository), `{Description}`, `{CloneURL.SSH}` and `{CloneURL.HTTPS}`
+currently there are `{OwnerName}` (name of repository), `{Description}`, `{CloneURL.SSH}` and `{CloneURL.HTTPS}`
```
Comment (6543) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]:
```suggestion
currently there are `{Name}` (name of repository), `{Description}`, `{CloneURL.SSH}`, `{CloneURL.HTTPS}` and `{OwnerName}`
```

**Review by 6543 — CHANGES_REQUESTED** [review]
just "apply"

**Review by 6543 — APPROVED** [review]
thx

**Comment by a1012112796** [comment]
I wander why this PR is not on any Milestone ?

**Comment by everhopingandwaiting** [comment]
on 1.12, but reopen

**Comment by lunny** [comment]
@a1012112796 most docs PRs we haven't added milestone because most of them are outdate documents so that we should not mark them on the milestone.

File: modules/repository/init.go [hunk 2]
```diff
@@ -55,6 +55,7 @@ func prepareRepoCommit(ctx models.DBContext, repo *models.Repository, tmpDir, re
 		"Description":    repo.Description,
 		"CloneURL.SSH":   cloneLink.SSH,
 		"CloneURL.HTTPS": cloneLink.HTTPS,
+		"OwnerName":      repo.OwnerName,
```
Comment (a1012112796) [inline modules/repository/init.go:58]: @lunny But  I also wander it's only a docs change ...

**Pull request merged**